MAUI/Cartesian-Charts/Line.md

```diff
@@ -267,3 +267,120 @@ this.Content = chart;
 
 ![Spline types chart in MAUI Chart](Chart-types_images/maui_spline_types_chart.png)
 
+### Enable Marker
+
+A marker, also known as a symbol, is used to determine or highlight the position of the data point. To give marker support in the series, set the ShowMarkers property to true. The default value of ShowMarkers property is false. 
```

Comment:
A marker, also known as a symbol, is used to determine or highlight the position of the data point. To enable markers in the series, set the [ShowMarkers]() property to true.